docs/modules/ROOT/pages/vshn-managed/postgresql/update-strategy.adoc

```diff
@@ -0,0 +1,48 @@
+= Instance Update Strategy
+
+Certain changes to the PostgreSQL instance spec requires a restart of the database to take effect.
+Most importantly changes to the `size` of the instance will need a restart of the instance to apply.§
+
+An instance's `.spec.parameters.updateStrategy` field allows you to configure and disable automated retarts after an upated of the PostgreSQL instance.
+There are two possible values:
+
+.Immediate
+
+The `Immediate` strategy will automatically restart the database if one is necessary for the updates to the instance to apply.
+This is the default update strategy.
+
+[IMPORTANT]
+Please be aware for single instance databases, this might cause a short downtime after updating.
```

Comment:
I'd argue that's also true for instance with multiple pods, as there will be some sort of failover via patroni. But I'd expect that to be much shorter.

Reply:
Fair. But I expect that to basically only be a connection reset. So I don't know if I'd actually call that "downtime". But you're right I'll rephrase.